lib/graphql/schema/unique_within_type.rb

```diff
@@ -21,7 +21,7 @@ def encode(type_name, object_value, separator: self.default_id_separator)
           raise "encode(#{type_name}, #{object_value_str}) contains reserved characters `#{separator}` in the type name"
         end
 
-        Base64.strict_encode64([type_name, object_value_str].join(separator))
+        Base64.urlsafe_encode64([type_name, object_value_str].join(separator))
```

Comment:
I think this should use `Base64Bp`? like the `decode` method does below

Reply:
I did see that, but I wasn't sure what the difference was or the reason for doing it. I can change it though 👍